BTVN10/main.js

```diff
@@ -0,0 +1,167 @@
+//b1:in bang nhan
+const getMultiply = (number) => {
```

Comment:
```suggestion
const getMultiply = number => {
```
Trong trường hợp hàm chỉ. nhận vào 1 params duy nhất thì có thể bỏ dấu `()` đi